regression_tests/stages/run_demodjango_smoke_tests.sh

```diff
@@ -28,3 +28,5 @@ export BASIC_AUTH_PASSWORD="$PASSWORD"
 
 echo -e "\nRun smoke tests"
 ./tests/browser/run.sh ${TARGET_ENVIRONMENT} smoke
+
+cd "${CODEBUILD_SRC_DIR}"
```

Comment:
so that main script isn't run from inside demodjango folder and can find maintenance page script